slices/gearman-job-server.yaml

```diff
@@ -0,0 +1,34 @@
+package: gearman-job-server
+
+essential:
+  - gearman-job-server_copyright
+
+slices:
+  bins:
+    contents: 
+      /usr/sbin/gearmand:
+
+  libs:
+    essential:
+        # gearman-job-server actually depends on adduser as well to create and
+        # remove a user during package install/removal - however we don't 
+        # support this currently so adduser is not added here.
```

Comment:
Let's move this comment to above the `essential` line to match style of other SDFs.
See `redis-tools.yaml` for an example.